### PR TITLE
Fix RpcNep5Tracker

### LINKED ISSUE #116: Update RpcNep5Tracker to work in NEO3
This plugin depends on UTXO, it must be refactored to work with NEO 3

From @shargon:
https://github.com/neo-project/neo-plugins/issues/114#issuecomment-518576175

### PR DESCRIPTION
**Pull request by erikzhang**
Fixes #116

### CODE REVIEW

File: RpcNep5Tracker/RpcNep5Tracker.cs [hunk 1]
```diff
@@ -67,10 +67,14 @@ private void ResetBatch()
         private void RecordTransferHistory(Snapshot snapshot, UInt160 scriptHash, UInt160 from, UInt160 to, BigInteger amount, UInt256 txHash, ref ushort transferIndex)
         {
             if (!_shouldTrackHistory) return;
+
+            Header header = snapshot.Height == 0
```
Comment (vncoelho) [inline RpcNep5Tracker/RpcNep5Tracker.cs]: Very good.

**Comment by shargon** [comment]
https://github.com/neo-project/neo-plugins/pull/125/files#diff-2c8a32803157f91fab2b7beedc206bb9R108 Now we call the `Transfer` event, no `MintTokens`, we should preserve it?

**Comment by superboyiii** [comment]
```getnep5balances``` works now. ```getnep5transfers``` still need to fix.
image

**Comment by erikzhang** [comment]
`getnep5transfers` only returns transactions in 7 days by default. You need to add parameters `startTime` and `endTime`. To get all transactions, you can just send the parameters:

```json
"params": ["AbT3X8KusycQKZiZBQhnHCuWFo1fpPuXWX", 0]
```

**Comment by superboyiii** [comment]
@erikzhang  I know this, but I make these transactions in two days which should display( in 7 days). It still doesn't work. For example, I use 2017/01/01 to be the start timestamp.
image

**Comment by superboyiii** [comment]
Works well now. Good job! 
image

**Pull request merged**